index.html

```diff
@@ -8,6 +8,10 @@
         <em>No upcoming events... <i class="fa fa-frown-o"></i></em>
     </div>
 </div>
+<h2 class="visible-xs"><i class="fa fa-map-marker"></i> Location</h2>
+<div class="visible-xs">
```

Comment:
Yeah makes sense. Was just a copy+paste from the sidebar.

Will update it when i get home